src/Sylius/Component/Core/composer.json

```diff
@@ -28,10 +28,12 @@
     "require": {
         "php": "^8.0",
         "enshrined/svg-sanitize": "^0.16",
+        "guzzlehttp/guzzle": "^6.5 || ^7.0",
```

Comment:
Same here.

Reply:
No idea, we've missed that while reviewing. As above – I'll try to catch Kevin later.

Reply:
I've dropped `php-http/guzzle6-adapter` however this dependency had `guzzlehttp/guzzle` dependency. After getting rid of an adapter, I had to add this package explicitly.